### PR TITLE
Uncomments WFPENG lines

### PR DESCRIPTION
**Pull request by cdobs**
HYPM WFPENG data can now be ingested (Redmine #12792 #15058) so those lines were uncommented

### CODE REVIEW

File: GI02HYPM/GI02HYPM_R00003_ingest.csv [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/CTDMO_sn*/*.hex,GI02HYPM-RIM01-02-CTDMOG039,recovered_inst,Available,
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-01-FLORDL000,recovered_wfp,Available,
+mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/C*.DA?,GI02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
 mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/A*.DA?,GI02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
```
Comment (rtravis-whoi) [inline GI02HYPM/GI02HYPM_R00003_ingest.csv:11]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: GS02HYPM/GS02HYPM_R00003_ingest.csv [hunk 2]
```diff
@@ -4,12 +4,12 @@ parser,filename_mask,reference_designator,data_source,status,notes
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/cg_data/MSIOC_sn/WFP/E22*.DA?,GS02HYPM-WFP03-00-WFPENG000,recovered_wfp,Available,lower profiler was lost - process data from the controller
 mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/cg_data/MSIOC_sn/WFP/E22*.DA?,GS02HYPM-WFP03-01-FLORDL000,recovered_wfp,Available,lower profiler was lost - process data from the controller
 mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/cg_data/MSIOC_sn/WFP/E22*.DA?,GS02HYPM-WFP03-03-DOSTAL000,recovered_wfp,Available,lower profiler was lost - process data from the controller
 mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/cg_data/MSIOC_sn/WFP/C22*.DA?,GS02HYPM-WFP03-04-CTDPFL000,recovered_wfp,Available,lower profiler was lost - process data from the controller
```
[the remark this answers is not in the document]
Comment (snwhite) [reply GS02HYPM/GS02HYPM_R00003_ingest.csv:15]: Looks good.

File: GP02HYPM/GP02HYPM_R00007_ingest.csv [hunk 3]
```diff
@@ -4,12 +4,12 @@ parser,filename_mask,reference_designator,data_source,status,notes
...
 mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn12997-01/0*/A*.DA?,GP02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
-#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/E*.DA?,GP02HYPM-WFP03-00-WFPENG000,recovered_wfp,Pending,Can't parse data. Redmine 12792
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/E*.DA?,GP02HYPM-WFP03-00-WFPENG000,recovered_wfp,Available,
 mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/E*.DA?,GP02HYPM-WFP03-01-FLORDL000,recovered_wfp,Available,
 mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/E*.DA?,GP02HYPM-WFP03-03-DOSTAL000,recovered_wfp,Available,
 mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/GWFP_sn13104-03/0*/C*.DA?,GP02HYPM-WFP03-04-CTDPFL000,recovered_wfp,Available,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply GP02HYPM/GP02HYPM_R00007_ingest.csv:15]: Looks good.

File: GP02HYPM/GP02HYPM_R00004_ingest.csv [hunk 4]
```diff
@@ -4,12 +4,12 @@ parser,filename_mask,reference_designator,data_source,status,notes
...
 mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/GWFP_sn13104-02/0*/A*.DA?,GP02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
-#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-00-WFPENG000,recovered_wfp,Pending,unable to parse. Redmine ticket 12792
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-00-WFPENG000,recovered_wfp,Available,
 mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-01-FLORDL000,recovered_wfp,Available,
 mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-03-DOSTAL000,recovered_wfp,Available,
 mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/GWFP_sn12936-02/0*/C*.DA?,GP02HYPM-WFP03-04-CTDPFL000,recovered_wfp,Available,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply GP02HYPM/GP02HYPM_R00004_ingest.csv:15]: Looks good.

File: GI02HYPM/GI02HYPM_R00007_ingest.csv [hunk 5]
```diff
@@ -4,7 +4,7 @@ parser,filename_mask,reference_designator,data_source,status,notes
...
 mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/CTDMO_sn*/*.hex,GI02HYPM-RIM01-02-CTDMOG039,recovered_inst,Available,
-#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Pending,unable to parse. Redmine ticket 12792
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Available,
 mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-01-FLORDL000,recovered_wfp,Available,
 mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
 mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/GWFP_sn*/0*/C*.DA?,GI02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply GI02HYPM/GI02HYPM_R00007_ingest.csv:10]: Looks good.

File: GI02HYPM/GI02HYPM_R00003_ingest.csv [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/cg_data/MSIOC_sn*/CTD/*.DAT,GI02HYPM-RIM01-02-CTDMOG039,recovered_host,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/CTDMO_sn*/*.hex,GI02HYPM-RIM01-02-CTDMOG039,recovered_inst,Available,
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-01-FLORDL000,recovered_wfp,Available,
+mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00003/instruments/GWFP_sn*/0*/C*.DA?,GI02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_R00003_ingest.csv:10]: Looks good.

File: GA02HYPM/GA02HYPM_R00003_ingest.csv [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GA02HYPM/R00003/instruments/GWFP_sn13104-05/0*/C*.DA?,GA02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GA02HYPM/R00003/instruments/GWFP_sn13104-05/0*/A*.DA?,GA02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GA02HYPM/R00003/instruments/GWFP_sn13104-04/0*/E*.DA?,GA02HYPM-WFP03-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GA02HYPM/R00003/instruments/GWFP_sn13104-04/0*/E*.DA?,GA02HYPM-WFP03-01-FLORDL000,recovered_wfp,Available,
+mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GA02HYPM/R00003/instruments/GWFP_sn13104-04/0*/E*.DA?,GA02HYPM-WFP03-03-DOSTAL000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GA02HYPM/R00003/instruments/GWFP_sn13104-04/0*/C*.DA?,GA02HYPM-WFP03-04-CTDPFL000,recovered_wfp,Available,
```
Comment (snwhite) [inline GA02HYPM/GA02HYPM_R00003_ingest.csv:15]: Looks good.

**Review by snwhite — APPROVED** [review]
All look good.

**Pull request merged**